README.md

```diff
@@ -22,9 +22,9 @@ compatibility matrix below.
 |    0.14.2\**    |                                       keptncontrib/prometheus-service:0.8.0                                       |
 |     0.15.1      |                                     keptncontrib/prometheus-service:0.8.1\***                                     |
 |     0.16.0      |                                     keptncontrib/prometheus-service:0.8.2\***                                     |
```

Comment:
Do you want to remove this line since Keptn `0.16.x` is compatible with `0.8.3`?